stylesheets/_modules.scss

```diff
@@ -2160,6 +2160,7 @@
   overflow: hidden;
   white-space: nowrap;
   text-overflow: ellipsis;
+  text-align: left;
```

Comment:
Can we make this `text-align: start`?

Reply:
We can (or not specify text alignmet at all), but it will align RTL text to the right. I'm not sure you want it to be aligned to the right:
![image](https://user-images.githubusercontent.com/4103710/64205070-a626a200-ce86-11e9-96e5-dbbdd0ff49b2.png)

Or not (but direction is set automatically to RTL):
![image](https://user-images.githubusercontent.com/4103710/64205279-1b927280-ce87-11e9-8ec1-65710387307c.png)

Reply:
We can leave it as you have it for now.